<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="G10CHLT003" name="com.arc.testcases.buildings.PaymentbyCCTest" tests="1" failures="0" timestamp="6 Apr 2017 11:37:56 GMT" time="18.579" errors="1">
  <testcase name="paymentbyCCTest" time="18.579" classname="com.arc.testcases.buildings.PaymentbyCCTest">
    <error type="java.lang.Error" message="Welcome Text is not correct expected [My Buildings] but found [My Projects]">
      <![CDATA[java.lang.Error: Welcome Text is not correct expected [My Buildings] but found [My Projects]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at com.arc.driver.CommonMethod.assertEqualsmessage(CommonMethod.java:230)
	at com.arc.ReusableMethods.ReusableMethodsLogin.assertPostLoginText(ReusableMethodsLogin.java:59)
	at com.arc.ReusableMethods.ReusableMethodsLogin.LoginToArc(ReusableMethodsLogin.java:109)
	at com.arc.testcases.buildings.PaymentbyCCTest.paymentbyCCTest(PaymentbyCCTest.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- paymentbyCCTest -->
</testsuite> <!-- com.arc.testcases.buildings.PaymentbyCCTest -->
